cpp/src/io/csv/csv_reader.cu

```diff
@@ -828,22 +832,22 @@ gdf_error allocateGdfDataSpace(gdf_column *gdf) {
 gdf_error launch_countRecords(raw_csv_t * csvData) {
 
 	int blockSize;		// suggested thread count to use
-	int minGridSize;	// minimum block count required
-	CUDA_TRY( cudaOccupancyMaxPotentialBlockSize(&minGridSize, &blockSize, countRecords) );
+ 	int minGridSize;	// minimum block count required
+ 	CUDA_TRY( cudaOccupancyMaxPotentialBlockSize(&minGridSize, &blockSize, countRecords) );
 
 	// Calculate actual block count to use based on bitmap count
-	// Each bitmap is for a 64-byte chunk, and each data index is bitmap ID * 64
-	int gridSize = (csvData->num_bits + blockSize - 1) / blockSize;
+ 	// Each bitmap is for a 64-byte chunk, and each data index is bitmap ID * 64
+ 	int gridSize = (csvData->num_bits + blockSize - 1) / blockSize;
```

Comment:
Since the previous PR was accepted, merge conflicts resulted in having to manually copy the previous changes back, hence the unintentional extra spaces. Will proceed to fix these trivial things.